entrypoint.sh

```
@@ -0,0 +1,71 @@
+  echo -e "${GREEN}Created markdownx widget template${NC}"
+fi
+
+# Create initial superuser if needed
+if [ "$CREATE_SUPERUSER" = "true" ]; then
+  echo -e "${GREEN}Creating superuser...${NC}"
+  python manage.py createsuperuser --noinput || true
+fi
+
+# Run custom command if provided
+if [ "$1" != "" ]; then
+  echo -e "${GREEN}Running custom command: $@${NC}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)